maint-lib/stagelib/replace.py

```diff
@@ -118,6 +118,11 @@ def do_variable_replace(replace_root_dir):
     logging.info('    Replacing variables')
     for dirname, subdirs, files in os.walk(replace_root_dir, topdown=True):
         for f in files:
+            # do not process the tarball
+            # this prevents the following error:
+            # UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
+            if "Sree-0.1.tar.gz" in f:
```

Comment:
If "Sree-" would be more generic and work across releases if you update it later